mesonbuild/minstall.py

```diff
@@ -756,6 +756,9 @@ def install_targets(self, d: InstallData, dm: DirMaker, destdir: str, fullprefix
                 self.set_mode(outname, install_mode, d.install_umask)
 
 def rebuild_all(wd: str) -> bool:
+    if (Path(wd) / 'meson-none-backend').is_file():
+        # nothing to build...
+        return True
```

Comment:
If the project is reconfigured with ninja, I think that file remains. Maybe instead of relying on a stamp file we should add the backend name into InstallData?

Reply:
Actually we already load Build object, it should contain coredata and thus the actual backend option. No need to make guesses here.

Reply:
We only do so if --no-rebuild is not specified.

Reply:
Done.